website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,146 @@
+
+To configure a `k8s` installation, simply open the folder of your installation and edit the `config-template/appsmith-configmap.yaml` file.
+
+Remove # before the variables to ensure they're not commented
+
+```
+// Example variable configuration format
+APPSMITH_GOOGLE_MAPS_API_KEY: "YOUR_API_KEY"
+```
+
+After making any changes, **remember to restart the pods** for the changes to take effect
+
+```
+// commands to restart k8s pods
+kubectl apply -f appsmith-configmap.yaml
+kubectl scale deployment appsmith-internal-server --replicas=0
+kubectl scale deployment appsmith-internal-server --replicas=1
+```
+
+## Configure Helm installations
+
+To configure environment variables using helm, follow these steps:
+
+<Tabs groupId="appsmithEditions">
+  <TabItem label="Community Edition" value="communityEdition"> 
+
+  You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen. 
+
+  * Alternatively, use a `values.yaml` file. You can specify the parameter values in the `applicationConfig` section. 
+
+```yaml
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
+  APPSMITH_FORM_LOGIN_DISABLED: ""
+  APPSMITH_SIGNUP_DISABLED: "true"
+```
+Once you have made changes to the values file, run the below command.
+
+```bash
+  helm upgrade --values values.yaml appsmith appsmith/appsmith
+```
+
+  </TabItem>
+  <TabItem label="Business Edition" value="businessEdition"> 
+
+   You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen. 
+
+  * Alternatively, use a `values.yaml` file. You can specify the parameter values in the `applicationConfig` section. 
+
+```yaml
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
+  APPSMITH_FORM_LOGIN_DISABLED: ""
+  APPSMITH_SIGNUP_DISABLED: "true"
+```
+Once you have made changes to the values file, run the below command.
+
+```bash
+  helm upgrade --values values.yaml appsmith appsmith-ee/appsmith
+```
+  </TabItem>
+</Tabs> 
+
+## Configure ECS installations
```

Comment:
This section has too much bold text. We've used bold text in above sections, but only for text we absolutely don't want our users to miss, and grab their attention towards it.

But in this section we seem to be using it for... several things. Emphasis on too many things, gives emphasis to nothing. Can we reduce the bold here please?

Reply:
Which parts need emphasis? I can make the changes.

Reply:
@sharat87 - I can do the above minor changes as part of this PR. On another note, we'll have to rewrite this page as for now it just focuses on Configuring Environment variables. However, it's more of a landing page that will give users an idea on:
- Introduction for Instance Configuration
- Things to be expected like custom domain, email config and more
- Configuring Environment variables for different installation types
- Troubleshooting section
- Further reading.
Do let me know your thoughts? I'll raise another ticket to look into this.

Reply:
Oh hey, sorry I thought this was all written by Sumesh, saw his name on the PR, so my comments were targeted towards him, didn't know you're working on this. 🙂

Reg. your question, honestly, I don't think anything in this section needs an emphasis. It's just fine being plain, but I'm not too strong on this. If you see something that should be bold, please go ahead. 🙂

Reply:
Sumesh has only added the configured Helm installations section and passed it on to me for rewriting. The others are not yet worked on. But it makes sense to fix the things as part of this PR as are no huge changes. 
I'll get the changes done, and we can merge this PR. We can discuss if the page needs restructuring and if needed can do a separate PR to accommodate that.